stdlib/runtime/string.gr

```diff
@@ -218,6 +301,8 @@ let rec heapValueToString = (ptr, extraIndents, toplevel) => {
         Memory.decRef(WasmI32.fromGrain(colspace))
         Memory.decRef(WasmI32.fromGrain(comlf))
 
+        Memory.free(fields) // Avoid double-free of record field names
```

Comment:
Why is this needed now but not before?

Reply:
`fields` is a value that didn't exist until this PR!